src/main/java/org/owasp/validator/css/CssScanner.java

```diff
@@ -96,6 +96,18 @@
 
     private static final Pattern p = Pattern.compile(CDATA, Pattern.DOTALL);
```

Comment:
While you are working on this, can you add a comment that describes what this variable is for/what it does? And maybe rename 'p' to something a bit more descriptive?

Reply:
Done